### DISCUSSION TITLE
Thoughts on a Slack or Discord channel?

### DISCUSSION THREAD

**Discussion by SimonMacIntyre** [mochi-mqtt/server#289] [General]
Not sure if this is jumping the gun at this stage, however I often have one-off questions that I'd love to ask without necessarily being worthy of a whole issue or discussion forum on Github! 

But I don't want to distract all you fine maintainers from your awesome work and productivity on bigger and better things than managing a slack/discord, so I understand the cons as well. 

An example if this was I was recently struggling to override the logger level to allow Log.Debug to propagate (I can only see Log.Error in my hooks), but I couldn't figure out how to override the servers log level. Thought it would have been a good quick one-off into a Discord general channel that someone might have been able to answer quickly, but felt like it wasn't worthy of a discussion especially since there is nothing wrong, it's just a noob(me) looking for a little guidance on something small.  (I tried `zerolog.SetGlobalLevel(zerolog.DebugLevel)` in my main.go but this did not do the trick. using Log.Info() is working and is fine, so I'm using that and its not a blocker, but it seemed like a learnable moment!)

In general I'd love to be able to have adhoc small chats or even just read other peoples chats to learn from them, stuff like that. 

Just an idea! I understand it is possibly a very selfish pursuit, so I respect you maintainers protecting your own time and being free from needless distractions, so maybe this is something for much later, and this is not the right time.

**Comment by dgduncan** [comment]
As per your issue with the logging level we have an example [here](https://github.com/mochi-mqtt/server/blob/main/examples/debug/main.go) on how to change the logging level for DEBUG.

As per my own personal thoughts on having a Slack, Discord, or IRC (I added that last one), I personally believe that it would not be very useful at this point in time for a few reasons, but mainly that I do believe the core maintainers may not want to set a precedent for live feedback or the expectation of that level of communication with users when it can not be promised.

At current state, I do believe the `Issues` component of Github works perfect for the type of asynchronous collaboration that we do; however, who knows what the future may hold :) but I do not speak for everyone.

**Reply by SimonMacIntyre** [reply]
Appreciate it, makes sense! Thanks for pointing me to that example, I thought I searched but not well enough! 🙏
